roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -9,6 +9,19 @@ class PackageVersion(NotContainerizedMixin, OpenShiftCheck):
     name = "package_version"
     tags = ["preflight"]
 
+    openshift_to_ovs_version = {
+        "3.6": "2.6",
+        "3.5": "2.6",
+        "3.4": "2.4",
+    }
+
+    openshift_to_docker_version = {
+        "3.1": "1.8",
+        "3.2": "1.10",
+        "3.3": "1.10",
```

Comment:
I think in origin the excluder work was never backported so it's only on the master branch (which makes sense).  You can safely assume the same docker version for the corresponding ocp version. eg, ocp 3.2 and origin 1.2 use docker 1.10.